centraldogma/src/main/java/com/linecorp/decaton/centraldogma/CentralDogmaPropertySupplier.java

```diff
@@ -145,19 +149,43 @@ public static CentralDogmaPropertySupplier register(CentralDogma centralDogma, S
         return new CentralDogmaPropertySupplier(centralDogma, project, repository, filename);
     }
 
+    public static CentralDogmaPropertySupplier register(CentralDogma centralDogma, String project,
+                                                        String repository, String filename,
+                                                        StaticPropertySupplier supplier) {
```

Comment:
I think in most cases people would use `StaticPropertySupplier` anyway, but we don't necessarily restrict this to be it, and just `PropertySupplier` (interface) is fine and more flexible?